src/ldf-client-ui.js

```diff
@@ -753,16 +753,19 @@ require('leaflet/dist/images/marker-shadow.png');
                 lat = feature.geometry.coordinates[1];
               }
               if (lon && lat) {
-                const marker = L.circleMarker([lat, lon]).addTo(self.map);
+                let marker = L.circleMarker([lat, lon]);
```

Comment:
Why is this change needed?

Reply:
If you add the last bit of addTo(self.map) it will be added to the map outside a layer so ideally we need to add the marker to a layer.

Then the let instead of const, Thought const makes it fixed throughout yet the marker value keeps changing as you add various points.